src/tests/api/DocumentSelectionNegativePath.test.ts

```diff
@@ -71,7 +71,7 @@ describe("Negative Path /userInfo Endpoint", () => {
 		// Post User Info
 		const userInfoResponse = await userInfoPost("Bearer " + constants.DEV_F2F_EXPIRED_ACCESS_TOKEN);
 		expect(userInfoResponse.status).toBe(400);
-		expect(userInfoResponse.data).toBe("Failed to Validate - Authentication header: Verification of exp failed"); 
+		expect(userInfoResponse.data).toBe("Failed to Validate - Authentication header: Failed to verify signature");
```

Comment:
Please check this with Zac - this is an expected test failure when running `npm run test:api` locally if you don't have an expired JWT to test against; but when run in pipeline this test passes as there is an expired JWT made available to the test via a github secret.

Reply:
reverted